godot-kotlin/godot-extension-injection/src/native/kotlin/godot/extension/injection/InjectionHelperPropertyHandler.kt

```diff
@@ -0,0 +1,17 @@
+package godot.extension.injection
+
+import godot.core.NodePath
+import godot.core.Variant
+import kotlin.reflect.KMutableProperty0
+
+class InjectionHelperPropertyHandler(private val nodePath: KMutableProperty0<NodePath>) {
+    fun get(): Variant {
+        return Variant.wrap(
+            nodePath.get()
```

Comment:
In this kind of case where you know the type of the instance (NodePath in this case) you should just write :
`nodePath.get().toVariant()`
Wrap is for a generic T Object so it's slower because we have to check the type before.